workflow/snakemake_rules/select_strains.smk

```diff
@@ -0,0 +1,135 @@
+'''
+This file contains rules that select strains for a build, extracts the sequences
+and the meta data subset.
+
+input:
+ - "data/lineage/metadata.tsv"
```

Comment:
In our discussion today, we want to support users who download separate segment FASTAs from GISAID. We may need to support files like `data/lineage/metadata_segment.tsv`